journal/src/test/java/io/camunda/zeebe/journal/file/SegmentedJournalTest.java

```diff
@@ -71,6 +70,8 @@ void shouldDeleteIndexMappingsOnReset() {
     // then
     assertThat(journal.getJournalIndex().lookup(journalIndexDensity)).isNull();
     assertThat(journal.getJournalIndex().lookup(2 * journalIndexDensity)).isNull();
+
+    journal.close();
```

Comment:
:x: Same as JournalReaderTest

Reply:
That'd require "leaking" the journal to an instance field so that it is accessible to and `AfterEach` callback

Reply:
I've gone with the first approach now